Atomic/atomic.py

```diff
@@ -272,6 +360,11 @@ def set_args(self, args):
             self.spc = False
 
         try:
+            self.system = args.system
+        except:
+            self.system = False
```

Comment:
This is not needed since this is defaulted above, Shouldn't we just pass as we have on other variable assignment.